vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1547,7 +1503,8 @@ public void fetchCurrentUser(@NotNull ModelCallback<User> callback, @Nullable St
      * @param callback The callback for the specific model type of the resource
      */
     @Nullable
-    public Call<Void> emptyResponsePost(String uri, @Nullable HashMap<String, String> postBody,
+    public Call<Void> emptyResponsePost(String uri,
+                                        @Nullable HashMap<String, String> postBody,
                                         VimeoCallback<Void> callback) {
```

Comment:
I added a follow up ticket to annotate all of the method parameters of VimeoClient:
https://vimean.atlassian.net/browse/ADF-199